docs/octopus-rest-api/octopus.migrator.exe-command-line/migrate.md

```diff
@@ -14,7 +14,7 @@ Where [<options>] is any of:
 
       --instance=VALUE       Name of the instance to use
       --file=VALUE           Octopus 2.6 (.octobak) file
-      --master-key=VALUE     Master Key used to decrypt the file
+      --master-key=VALUE     Master key used to decrypt the file
```

Comment:
Looks like this will be fixed in 2020.2.0 https://github.com/OctopusDeploy/OctopusDeploy/commit/a92ef7008f305a1eee65eb82c252cea00b022785

Reply:
Awesome. I'll this open until then to act as a reminder.